frameworks/Swift/vapor/Sources/TfbCommon/World.swift

```diff
@@ -0,0 +1,10 @@
+import Random
+
+public struct WorldMeta {
+  private init() { }
+
+  public static let maxId: UInt32 = 10000
+  public static let randomId = { () -> UInt32 in UInt32(1) + CryptoRandom.uint32 % maxId }
```

Comment:
@tanner0101 I agree. `PseudoRandom` fits better. Will change this a bit later.

Looks like `CryptoRandom` uses `RAND_bytes` and `PseudoRandom` uses `RAND_pseudo_bytes` and these functions have same implementation in clibressl 🙃